config/testdata/http.conf.tlsconfig.good.yml

```diff
@@ -0,0 +1,294 @@
+proxy_url: "http://remote.host"
+tls_config:
+  # this is config/testdata/tls-ca-chain.pem
+  ca: !!binary |
```

Comment:
not sure where or if to document this bit: the `!!binary` tag causes the YAML library to treat the contents of the field as base64-encoded. The code itself will never see the base64-encoded version, only the decoded payload.